backend/main.py

```
@@ -11,208 +17,180 @@
-            root_dir = os.path.dirname(current_dir)
-            ai_dir = os.path.join(root_dir, 'ai')
-            
-            if ai_dir not in sys.path:
-                sys.path.append(ai_dir)
-                
+            ai_dir = os.path.join(os.path.dirname(current_dir), 'ai')
+            if ai_dir not in sys.path: sys.path.append(ai_dir)
+
             from pipeline.pipeline import GlaucomaPipeline
-            
             global glaucoma_pipeline
```

Ruff (0.15.13)

[warning] Using the global statement to update `glaucoma_pipeline` is discouraged

(PLW0603)